cord/orm/dataset.py

```diff
@@ -166,14 +166,24 @@ class ImageGroupOCR:
 
 
 @dataclasses.dataclass(frozen=True)
-class ReEncodeVideoTaskResult:
+class ReEncodeVideoTaskResult(Formatter):
     data_hash: str
     signed_url: str
     bucket_path: str
 
+    @classmethod
+    def from_dict(cls, json_dict: Dict):
+        return ReEncodeVideoTaskResult(json_dict["data_hash"], json_dict["signed_url"], json_dict["bucket_path"])
```

Comment:
is there a way to avoid writing from_dict method for this?
the field names are exactly same as the name in the input dict